packages/dx-react-scheduler-material-ui/src/templates/date-navigator/calendar/navigator.jsx

```diff
@@ -25,19 +25,20 @@ const NavigatorBase = ({
     {...restProps}
   >
     <NavigationButton
-      back
+      type="back"
```

Comment:
Can we use constants instead of string variables here?

Reply:
I'm not sure. The consts will look like `const BACK = 'back'; const FORWARD = 'forward';` I think it doesn't make sense. BTW, we [don't use](https://github.com/DevExpress/devextreme-reactive/blob/master/packages/dx-grid-core/src/plugins/integrated-sorting/computeds.js#L24) such consts for sorting directions.

Reply:
Okay 👌🏽